doc/UsersGuide/source/InputsOutputs.rst

```diff
@@ -79,19 +79,19 @@ The static input files are listed and described in :numref:`Table %s <FixFiles>`
 ---------------------------------------------
 Grid description and initial condition files
 ---------------------------------------------
-The input files containing grid information and the initial conditions are listed and described in :numref:`Table %s <GridICFiles>`.
+The input files containing grid information and the initial conditions for global configurations are listed and described in :numref:`Table %s <GridICFiles>`.  The input files for a limited area model (LAM) configuration including grid information and initial and lateral boundary conditions are listed and described in :numref:`Table %s <RegionalGridICFiles>`.  Note that the regional grid is referred to as Tile 7 here, and are generated by several pre-processing utilities.
```

Comment:
When defining an acronym, you must capitalize the first letter of each word, as in a title. Also, "limited-area" should be hyphenated because it is being used as an adjective.

> Limited-Area Model (LAM) configuration

Also, replace "and...and" with a comma-separated series. The sentence ran away from itself; it's better to rewrite it in a more simple manner like:

> The input files for a Limited-Area Model (LAM) configuration includes initial and boundary conditions, and grid information (see `:numref:` Table...)

Also, this "are" should be "is"

> Note that the regional grid is referred to as Tile 7 here, and **is** generated by several pre-processing utilities